integrations/adobe-analytics/lib/index.js

```diff
@@ -10,7 +10,8 @@ var each = require('@ndhoule/each');
 var iso = require('@segment/to-iso-string');
 var Track = require('segmentio-facade').Track;
 var trample = require('@segment/trample');
-
+const analytics = require('@segment/analytics.js-core');
```

Comment:
```suggestion
var analytics = require('@segment/analytics.js-core');
```

`const` not yet available in analytics.js 1.0.